docs/section10/10-1.md

```diff
@@ -47,21 +43,13 @@ If humidification was modeled in the proposed for a specific zone, check if it w
     Check if zone was modeled with humidification in the B_RMR 
     - Get analogous zone in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
     - Get list of hvac systems that serve this zone in the B_RMR, should only be one, note that this RDS assumes only one: `hvac_list_b = get_list_hvac_systems_associated_with_zone(B_RMR,zone_b.id)`
-    - Get the baseline system type (used to determined whether to check the preheat or heating coils in hvac_b for humidification): `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
-    - Check if the baseline system type would include a preheat coil, if so then get the humidification_type for the preheat coil : `if hvac_sys_type_b in ["SYS-5","SYS-5b","SYS-6", "SYS-6b","SYS-7","SYS-7a","SYS-7b","SYS-7c","SYS-8","SYS-8a","SYS-8b","SYS-8c"]:`
-        - Check if the humidification_type equals Null for the preheat coil, if it does then set the type equal to None: `if hvac_b.preheat_system[0].humidification_type = Null: humidification_type_b = "None"`
-        - Else (i.e. it does not equal Null), get humidification type for hvac_b: `Else: humidification_type_b = hvac_b.preheat_system[0].humidification_type`
-    - Else, if the baseline system type would not include a preheat coil then check the heating coil: `Else:`
-        - Check if the humidification_type equals Null for the heating coil, if it does then set the type equal to None: `if hvac_b.heating_system[0].humidification_type = Null: humidification_type_b = "None"`
-        - Else (i.e. it does not equal Null), get humidification type for hvac_b: `Else: humidification_type_b = hvac_b.heating_system[0].humidification_type`
+    - Check if the humidification_type equals Null for the HVAC system, if it does then set the type equal to None: `if hvac_b.humidification_type = Null: humidification_type_b = "None"`
```

Comment:
`hvac_b `is not defined.  I assume it is equal to `hvac_list_b[0]`?

Reply:
Good catch. Revised.